make-release.sh

```diff
@@ -302,13 +325,20 @@ verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-machine-exec:${
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-devfile-registry:${CHE_VERSION} 60
 # shellcheck disable=SC2086
 verifyContainerExistsWithTimeout ${REGISTRY}/${ORGANIZATION}/che-dashboard:${CHE_VERSION} 60
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/che-incubator/configbump.git" ${BRANCH}
+# shellcheck disable=SC2086
+verifyBranchExistsWithTimeout "https://github.com/eclipse/che-jwtproxy.git" ${BRANCH}
```

Comment:
should this instead be a call to `check-branches.sh` ?

Reply:
rewrote, now both scripts use the shared function from util.sh